README.md

```diff
@@ -46,21 +92,23 @@ This software is distributed under the [MIT License](./LICENSE).
 
 ⚠️ Drawing visualizations is expensive and can impact the scheduling of audio threads in surprising ways. You can observe behavior without the confounding effect of drawing by briefly turning visualizations off, performing a test, and then turning them back on.
 
-## Load
+## Visualizations
+
+### Load
```

Comment:
Now that there is much more text above this section you have to scroll to actually see the UI element each paragraph refers to. Some of the UI elements, such as "Work distribution" are collapsed in the screen shot, so you don't see what is being talked about until you build and run the app.

Might be nice to have a picture of each UI element before each paragraph. You'd then get a better idea about what the app does before downloading and building it.

Reply:
I agree and have thought about this, but I don't want to have to chop up images to change the UI, so I decided against it . Updating the single screen screen is already enough work.

Reply:
Fair enough.